codespell_lib/data/dictionary.txt

```diff
@@ -20093,8 +20126,8 @@ sate->state, sate,
 satelite->satellite
 satelites->satellites
 satelitte->satellite
-Saterday->Saturday
-Saterdays->Saturdays
+saterday->saturday
```

Comment:
Saturday is a proper noun so I think the RHS should start with a capital.

Reply:
b9992e9